database/postgres/postgres.go

```diff
@@ -340,12 +340,28 @@ func (p *Postgres) ensureVersionTable() (err error) {
 	defer func() {
 		if e := p.Unlock(); err == nil {
 			err = e
+		} else if e != nil {
+			err = &database.Error{OrigErr: err, Err: e.Error()}
 		}
 	}()
 
-	query := `CREATE TABLE IF NOT EXISTS "` + p.config.MigrationsTable + `" (version bigint not null primary key, dirty boolean not null)`
+	// We don't use a simple CREATE IF NOT EXISTS below because we want to support
```

Comment:
Thanks for adding this documentation!

I'm fine dropping support for versions of postgres below 9.4 since they're not officially supported: https://www.postgresql.org/support/versioning/

I still need to update the postgres versions we test against...

Reply:
OK. Let's keep it this way and once you update the versions of Postgres we test against, feel free to convert this to a CREATE IF NOT EXISTS.

Reply:
OK, I [just checked](https://github.com/golang-migrate/migrate/blob/master/database/postgres/postgres_test.go#L28) and it seems we no longer test with 9.0. I'll just revert this to use CREATE IF NOT EXISTS then.